src/js/main.js

```diff
@@ -0,0 +1,81 @@
+export const DiscourseRADParser = () => {
+  const radTabsElements = document.querySelectorAll('.rad-tabs-element');
+
+  radTabsElements.forEach((radTabsElement) => {
+    setDefaults(radTabsElement);
+    setupDropdowns(radTabsElement);
+  });
+
+  toggleElements();
+}
+
+const setDefaults = radTabsElement => {
+  const dropdowns = radTabsElement.querySelectorAll('select');
+
+  dropdowns.forEach((dropdown) => {
+    const name = dropdown.name;
+
+    const options = dropdown.querySelectorAll('option');
+    let defaultWasSet = false;
+    options.forEach((option) => {
+      if (localStorage.getItem(name) === option.value) {
+        defaultWasSet = true;
+        option.setAttribute('selected', 'selected');
+      }
+    });
+
+    if (!defaultWasSet) {
+      const firstOption = dropdown.firstElementChild;
+      firstOption.setAttribute('selected', 'selected');
+
+      if (localStorage.getItem(name) === null) {
+        localStorage.setItem(name, firstOption.value);
+      }
+    }
+  });
+}
+
+const setupDropdowns = radTabsElement => {
+  const dropdowns = radTabsElement.querySelectorAll('select');
+
+  dropdowns.forEach(function (dropdown) {
+    attachDropdownEvents(dropdown);
+  });
+}
+
+const attachDropdownEvents = dropdown => {
+  dropdown.addEventListener('change', function (e) {
+    const newValue = e.target.value;
+    const dropdownName = dropdown.name;
+
+    localStorage.setItem(dropdownName, newValue);
+
+    const dropdowns = document.querySelectorAll(`select[name=${dropdownName}][class*=rad-dropdown-element]`);
```

Comment:
Might be better to loop the orginal set `radTabsElements` and use `classList.contains`

Reply:
Sooo... I made some changes here so that I avoid using this complex select query.

The thing I am doing here is the following:
- I have 3 UI/CLI dropdowns on the page
- I switch one of them from UI to CLI
- I must update all the UI/CLI dropdowns to CLI
- Other dropdowns should not be affected

Here I am trying to fetch all the UI/CLI dropdowns that have the same name and set them to CLI.